src/Network/Network/FirewallPolicy/NewAzureApplicationGatewayFirewallPolicyCommand.cs

```diff
@@ -53,11 +53,23 @@ public class NewAzureApplicationGatewayFirewallPolicyCommand : ApplicationGatewa
         public virtual string Location { get; set; }
 
         [Parameter(
-             Mandatory = false,
-             ValueFromPipelineByPropertyName = true,
-             HelpMessage = "The list of CustomRules")]
+         Mandatory = false,
+         ValueFromPipelineByPropertyName = true,
+         HelpMessage = "The list of CustomRules")]
         public PSApplicationGatewayFirewallCustomRule[] CustomRule { get; set; }
 
+        [Parameter(
+         Mandatory = false,
+         ValueFromPipelineByPropertyName = true,
+         HelpMessage = "Policysettings of the firewall policy")]
+        public PSApplicationGatewayFirewallPolicySettings PolicySettings { get; set; }
```

Comment:
Per PowerShell guidelines, parameter names should be in siingular form. But not sure about this case. @markcowl, please take a look?